src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -149,10 +158,32 @@ local function tabulateCommon( bPaged, ... )
 end
 
 function tabulate( ... )
+    tArgs = { ... }
```

Comment:
Please remove these checks, and the similar ones in pagedTabulate, and put checks in tabulateCommon with an error level of 3 instead.